.github/ISSUE_TEMPLATE/documentation_report.yml

```diff
@@ -0,0 +1,115 @@
+      *Complete **all** sections as described, this form is processed automatically.*
+
+      [issue search]: https://github.com/ansible/ansible/search?q=is%3Aissue&type=issues
+
+
+- type: textarea
+  attributes:
+    label: SUMMARY
+    description: |
+      Explain the problem briefly below, add suggestions to wording or structure.
+
+      **HINT:** Did you know the documentation has an `Edit on GitHub` link on every page?
+    placeholder: >-
+      I was reading the ansible-core documentation of version X and I'm having
+      problems understanding Y. It would be very helpful if that got
+      rephrased as Z.
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: ISSUE TYPE
+    # FIXME: Once GitHub allows defining the default choice, update this
+    options:
+    - Documentation Report
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: COMPONENT NAME
+    description: >-
+      Write the short name of the rst file, module, plugin, task or
+      feature below, *use your best guess if unsure*.
+    placeholder: docs/docsite/rst/dev_guide/debugging.rst
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: ANSIBLE VERSION
+    description: >-
+      Paste verbatim output from `ansible --version` between
+      tripple backticks.
+    value: |
+      ```console (paste below)
+      $ ansible --version
+
+      ```
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: CONFIGURATION
+    description: >-
+      Paste verbatim output from `ansible-config dump --only-changed` between quotes.
+    value: |
+      ```console (paste below)
+      $ ansible-config dump --only-changed
+
+      ```
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS / ENVIRONMENT
+    description: >-
+      Provide all relevant information below, e.g. OS version,
+      browser, etc.
+    placeholder: Fedora 33, Firefox etc.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: ADDITIONAL INFORMATION
+    description: |
+      Describe how this improves the documentation, e.g. before/after situation or screenshots.
+
+      **Tip:** It's not possible to upload the screenshot via this field directly but you can use the last textarea in this form to attach them.
+
+      **HINT:** You can paste https://gist.github.com links for larger files.
+    placeholder: >-
+      When the improvement is applied, it makes it more straightforward
+      to understand X.
+  validations:
+    required: true
+
+
+- type: checkboxes
+  attributes:
+    label: Code of Conduct
+    description: |
+      Read the [Ansible Code of Conduct][CoC] first.
+
+      [CoC]: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html?utm_medium=github&utm_source=issue_form--documentation_report.yml
+    options:
+    - label: I agree to follow the Ansible Code of Conduct
+      required: true
```

Comment:
@mattclay are you talking about this :point_up:? Would it be better to add it in a separate PR or are you opposed to the idea in general?

Reply:
Yes, it's not part of the current templates.

I wouldn't open a separate PR to add it either. The issue templates should contain the minimum text and prompts necessary to facilitate reporting issues. We should review them periodically to see if there is anything we can remove or clarify to encourage better issue reporting. For example:

* The documentation issue probably doesn't need the "OS / Environment" section. I believe @bcoca already mentioned this elsewhere.
* With the way these templates work, the "Additional Information" section is redundant with the comment section that immediately follows.

Reply:
> * With the way these templates work, the "Additional Information" section is redundant with the comment section that immediately follows.

WSYWIG is only enabled for one form, for screenshots. It's non-enforceable.

> I wouldn't open a separate PR to add it either.

I checked with @gundalow if it's a good idea to have the CoC check and he thinks it is.